index.d.ts

```diff
@@ -0,0 +1,392 @@
+/**
+  The login function will login
+
+  @param {object} [options] - Options object for the deploy function
```

Comment:
Should this be options for the `login` function?

Reply:
:facepalm: